css/css-text/letter-spacing/letter-spacing-cursive-001.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<head>
+<meta charset="utf-8">
+<title>
+CSS Text Test - Letter-spacing should not be applied for Cursive Scripts.
+</title>
+<link rel="author" title="Sejal Anand" href="mailto:sejalanand@microsoft.com">
+<link rel="help" href="https://www.w3.org/TR/css-text-3/#cursive-tracking">
+<link rel="match" href="reference/letter-spacing-cursive-001-ref.html">
+<meta name="assert" content="Letter-spacing must not be applied to Arabic text,
+ preserving cursive connections.">
+<style>
+div {
+  margin: 1em;
+  font: 24px serif;
+  white-space: nowrap;
+  display: inline-block;
+  outline: 1px solid gray;
+}
+.letterspacing {
+  letter-spacing: 10px; /* Should NOT affect Arabic text */
+}
+</style>
+</head>
+<body>
+<p>
+The following test will pass if no spacing appears between letters within
+Arabic words. Both Arabic texts should display identically.
+</p>
+<div class="letterspacing">
+<span>١. مرحباً</span>
+</div>
+<br>
+<div>
+<span>٢. مرحباً</span>
```

Comment:
If we want both pieces of text to render identically (regardless of letter spacing), they should actually be the same text.

```suggestion
<span>١. مرحباً</span>
```

Reply:
done